### PR TITLE
Update index.md

### PR DESCRIPTION
**Pull request by xcloudscript**
Adoption

### CODE REVIEW

**Review by cpanato — CHANGES_REQUESTED** [review]
please sign the dco

**Review by imjasonh — APPROVED** [review]

File: docs/index.md [hunk 1]
```diff
@@ -32,6 +32,7 @@ This can make it a good fit for lightweight CI/CD use cases.
 - [Sigstore](https://sigstore.dev)
 - [Shipwright](https://shipwright.io)
 - [Capsule](https://capsule.clastix.io/)
+- [ClousScript](https://cloudscript.com.br/)
```
Comment (imjasonh) [inline docs/index.md:35]:
```suggestion
- [CloudScript](https://cloudscript.com.br/)
```

**Review by imjasonh — CHANGES_REQUESTED** [review]

**Review by cpanato — DISMISSED** [review]
if jason approved i will set the check

**Review by imjasonh — APPROVED** [review]

**Review by cpanato — APPROVED** [review]

**Comment by xcloudscript** [comment]
Thanks e and congratulations the project is really nice.

**Pull request merged**